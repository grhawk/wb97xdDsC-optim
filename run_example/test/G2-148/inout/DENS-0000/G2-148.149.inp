 $DATA
 Should be setted!
 C1
 H     1.00     0.000000     0.000000     0.000000
 $END
 $DFT NRAD=24 NLEB=110 DDSC=.t. $END
 $SCF DIRSCF=.t. $END
 $BASIS NGAUSS=6 GBASIS=N31 $END
 $CONTRL DFTTYP=B97 ICHARG=0 EXETYP=RUN MULT=2 MAXIT=200 SCFTYP=UHF 
RUNTYP=ENERGY $END
 $SYSTEM MWORDS=8 $END