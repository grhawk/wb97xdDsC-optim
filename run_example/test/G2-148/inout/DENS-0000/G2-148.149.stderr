/var/spool/slurm/job12428/slurm_script: line 9: module: command not found
